.github/ISSUE_TEMPLATE/Release_23.12_FOSS_ComplianceTemplate.md

```diff
@@ -0,0 +1,80 @@
+---
+name: Release Compliance 23.12
+about: Use this template for tracking Catena-X Release 23.12.
+title: "[FOSS NAME] Release Compliance"
+labels: compliance
+assignees: @kelaja
+milestones: 23.12
+---
+
+<!-- 
+Thanks for your contribution! Please fill out this template as good as possible. 
+Important: Contributing Guidelines can be found here: https://eclipse-tractusx.github.io/docs/oss/how-to-contribute
+Checkout the repository README for process description. 
+-->
+
+
+
+## Release Compliance 23.12
+Source in Catena-X Confluence + Expert Contacts [here](https://confluence.catena-x.net/x/DOZkBQ).
+
+
+ - [ ] **Gaia-X compliance** confirmed by Catena-X expert.
+	
+    - Obtain approval from CX Gaia-X expert(s), prior to Gate review.
+
+    - Provide info as early as possible; expect optimization loops based on feedback
+
+    => check [linked PDF](https://confluence.catena-x.net/download/attachments/90498572/SD_Data_for_Onboarding.pdf?version=1&modificationDate=1690457195303&api=v2) summary for requirements
+
+- [ ] **GDPR compliance** must be ensured
+  (personal data; Data Protection & Privacy DPP)
+  - assessment must be completed via Catena-X [GDPR questionnaire](https://confluence.catena-x.net/download/attachments/90498572/Catena-X%20GDPR%20Declaration%20and%20Requirements_V3.xlsx?version=1&modificationDate=1690457195339&api=v2)
+  - either the product/service doesn`t process any relevant data,
+  - or - in case the assessment reveals potential issues - a more severe "Data Privacy Impact Assessment" shall be conducted (if applicable: robust mitigation actions must be completed)
+  Provide evidence for GDPR compliance prior to Gate review. Previous assessments remain valid, as long as approach to handling of personal data was not changed.
+
+- [ ] **Interoperability Check**
+
+  Acceptance criteria are fulfilled as described in in the following area on confluence: [Interoperability Quality Gate Requirements](https://confluence.catena-x.net/x/DkwjB)
+
+  - Approval will be documented centrally by Expert(s)
```

Comment:
is this still true? Or would the Experts leave their optimization requests as well as final approval within the GitHub issue?

Reply:
I will add that the final approval should be documented in the GitHub Issues, then its on the Experts to make the central documentation by them selfs .